src/components/SearchModal/styleds.tsx

```diff
@@ -34,7 +34,7 @@ export const MenuItem = styled(RowBetween)<{ redesignFlag?: boolean }>`
     background-color: ${({ theme, disabled, redesignFlag }) =>
       (redesignFlag && theme.hoverDefault) || (!disabled && theme.deprecated_bg2)};
   }
-  opacity: ${({ disabled, selected }) => (disabled || selected ? 0.5 : 1)};
+  opacity: ${({ disabled, selected, dim }) => (dim ? 0.4 : disabled || selected ? 0.5 : 1)};
```

Comment:
why .4 for dim and .5 for disabled? seems very minimally different

Reply:
i would also call it something other than dim. it's basically "blocked" right?

Reply:
sounds good done.

Reply:
yeah tbh.... i think we can just set both to 0.4. changing